server/authorization.go

```diff
@@ -74,7 +75,13 @@ func (a *AuthorizationServer) Register(s *grpc.Server, handler *Handler) {
 // Check does check
 func (a *AuthorizationServer) Check(ctx gocontext.Context, req *authv3.CheckRequest) (*authv3.CheckResponse, error) {
 	if !a.handler.Ready() {
-		return a.unavailable(req), nil
```

Comment:
After reading this, I tend to agree that the prior versions were more readable. I don't think it's required, but if you wanted to bring them back, I don't think that was an essential part of the original design.

Reply:
I think with the call-site documentation in place, having a single handleFault looks good too.

Reply:
If the others are fine with is, I am too. Thanks for taking a look!